xml/release-notes.xml

```diff
@@ -95,6 +95,44 @@
    Make sure to also review <xref linkend="sec.driver"/>.
   </para> -->
 
+  <sect2 xml:id="installation-new-update-repos">
+   <title>&opensuseleap; has newly three update repos</title>
+   <!-- boo#1186593 -->
+   <para>
+    Since &opensuseleap; 15.3 the maintenance setup consists of three main update repositories.
+    These are repo-update, repo-backports-update, and repo-sle-update. The latter two are brand new and
+    are a result of reworking distribution to re-use binaries from SUSE Linux Enterprise.
+
+    These repositories, are available and checked during the Online installation of &opensuseleap;.
+    We would like to kindly ask users not to uncheck them.
+    New update repo definitions for &opensuseleap; 15.3 will be additionally supplied via a 0day maintenance update of the <package>openSUSE-release</package>.
+
+    The update will be delivered via the traditional maintenance channel (repo-update).
+    It will carry a special update flag, to signalize that it touches the software management area which is then specially handled by zypp.
```

Comment:
```suggestion
    It will carry a special update flag that means it touches the software management area which is then specially handled by zypper.
```